chart/kubeapps/README.md

```diff
@@ -178,6 +178,7 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
 | `dashboard.customStyle`                           | Custom CSS injected to the Dashboard to customize Kubeapps look and feel                  | `""`                 |
 | `dashboard.customComponents`                      | Custom Form components injected into the BasicDeploymentForm                              | `""`                 |
 | `dashboard.customLocale`                          | Custom translations injected to the Dashboard to customize the strings used in Kubeapps   | `""`                 |
+| `dashboard.defaultTheme`                          | Default theme used in the Dashboard if the user has not selected any theme yet.           | `""`                 |
```

Comment:
Just for letting you know, we are using this tool for generating the readme automatically: https://github.com/bitnami-labs/readme-generator-for-helm

Reply:
Ah I missed that, thanks!